tsconfig.json

```diff
@@ -1,14 +1,21 @@
 {
   "compilerOptions": {
-    "target": "es2018",
+    "target": "es2021",
     "module": "commonjs",
-    "strict": true,
+    "strict": false,
```

Comment:
ideally we don't commit this change to `strict`

Reply:
sure. will fix with more strict typing and revert this change.